editor/editor_export.cpp

```diff
@@ -1803,6 +1803,40 @@ bool EditorExportPlatformPC::can_export(const Ref<EditorExportPreset> &p_preset,
 	valid = dvalid || rvalid;
 	r_missing_templates = !valid;
 
+	if (get_os_name() == "Windows") {
```

Comment:
What I meant with my previous review comment is that this code should be run on macOS and Linux too. However, on all platforms, this code should only run on Windows *export presets*, not on macOS or Linux export presets.

Reply:
> What I meant with my previous review comment is that this code should be run on macOS and Linux too, as you can export Godot projects for Windows from other platforms. (Apologies if this wasn't clear before.)
> 
> However, on all platforms, this code should only run on Windows _export presets_, not on macOS or Linux export presets.

Thanks for your review and I think it was clear enough. I also believe that this code can work as expected. However, I do need to move it from `EditorExportPlatformPC` to `EditorExportPlatformWindows` since it isn't needed on macOS or Linux.